rollup.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 
 info:
   title: Cartesi Rollup HTTP API
-  version: 0.8.0
+  version: 0.8.1
```

Comment:
0.9 would be better?

Reply:
Should we revisit this point now that fields got changed?